src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
Please consider the use of try/catch for a better error handling

https://medium.com/@sudhirkmr529/mastering-error-handling-with-async-await-in-javascript-cb1c877bb554